models/studentDataModel.js

```diff
@@ -0,0 +1,38 @@
+/* dataModel.js
+*/
+​
+var mongoose = require('mongoose');   
+​
+var dataSchema = mongoose.Schema({
+	type: String,
+	date: Date,
+	data: Array
+});  
+​
+var studentSchema = mongoose.Schema({
+	id: Number,
+	name: String,
+	program: String,
+	attendance: [{type: Schema.ObjectId, required: false, ref: 'attendance'}],
+	grades: [{type: Schema.ObjectId, required: false, ref: 'grades'}],
+	data: [{type: Schema.ObjectId, required: false, ref: 'data'}],
+	archived: {type: Boolean, required: false}
+});
+
+var userSchema = mongoose.Schema({
+	username: String,
+	password: String
+});
+​
+/*
+{
+	type: 'attendace'
+	date: 'Jul 01 2016'
+}
+*/
+​
+module.exports.attendace = mongoose.model('attendace', dataSchema); 
```

Comment:
I think convention is to [capitalize schema names](http://mongoosejs.com/docs/populate.html)